sdk/src/main/resources/account/rpc/api/eth_openrpc_v2.json

```diff
@@ -0,0 +1,2415 @@
+{
```

Comment:
I have marked every function that we will not implement, these should be removed. Everything else is already implemented or should be at some point.